### DISCUSSION TITLE
RuntimeError: unexpected EOF, expected 9378161 more bytes. The file might be corrupted.

### DISCUSSION THREAD

**Discussion by suforme** [milvus-io/milvus#19089] [Q&A and General discussion]
%%time
collection = create_milvus_collection('reverse_image_search', 2048)

dc = (
    towhee.glob['path']('train/*/*.JPEG')
      .image_decode['path', 'img']()
      .image_embedding.timm['img', 'vec'](model_name='resnet50')
      .ann_insert.milvus[('path', 'vec'), 'mr'](collection=collection)
)

RuntimeError: unexpected EOF, expected 9378161 more bytes. The file might be corrupted.

**Comment by yhmo** [comment]
Seems this error is thrown from towhee, during decoding the image files. Could you provide more details?

**Reply by suforme** [reply]
![image](https://user-images.githubusercontent.com/18099778/189019521-d31ff078-1e2c-4748-a3b5-a98c244e0ca9.png)

**Reply by suforme** [reply]
The likely reason is that there is an error in downloading the model, and the download file needs to be deleted, but I don't know where the download file path. Is there any other way to download and load the model.

**Reply by junjiejiangjjj** [reply]
cd  ~/.cache/torch/hub/checkpoints/ 
ls -larth
you will find some models:
![image](https://user-images.githubusercontent.com/14136703/189022481-06fe484f-55a8-4f12-8fcf-c9c25f1f9bdd.png)
rm the resnet50 model, and rerun your code.

**Reply by suforme** [reply]
where is the path in windows10?

**Reply by suforme** [reply]
get it, thanks

**Reply by jaelgu** [reply]
@suforme After rerun, you can check the size of resnet50 checkpoint file under `~/.cache/torch/hub/checkpoints/`. The correct size should be about 98MB. If the checkpoint file size is too small, it means incomplete download and will result in unexpected EOF.
